example/example-domain/src/main/java/uk/gov/justice/services/example/cakeshop/domain/Ingredient.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.services.example.cakeshop.domain;
+
+public class Ingredient {
```

Comment:
Does this need an @Event annotation too? or are we ignoring it for the moment as it isn't implemented.. just wondering as we specified the POJO.